choco/nunit-console-with-extensions.nuspec

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-console-with-extensions</id>
+    <version>0.0.0</version>
```

Comment:
> This is how chocolatey generates templates, so it's at least their preferred approach.

I'm not sure what version of Chocolatey you are on, but this hasn't been the way that Chocolatey generates templates for over a year. Current version is 0.10.7. and it separates package specifics from software specifics now in generation.